_layouts/default.html

```diff
@@ -18,6 +18,8 @@
 <div class="container">
 
     <div id="navigation">
+        <!-- Integrated Searh Box from docdoc Go -->
```

Comment:
Den Kommentar bitte noch einmal überarbeiten. Mit folgender Syntax lassen sich auch Kommentare schreiben, die nicht in der HTML-Seite landen. Damit könnte man hier noch mehr Hinweise für Entwickler erfassen.

```
{% comment %}
this is commented out
{% endcomment %}
```